building.sk

```diff
@@ -10,6 +10,8 @@ unchanged building blocks:
 
 	# Sediments
 	dirt [block¦s] = minecraft:dirt
+	coarse dirt [block¦s] = minecraft:coarse_dirt
```

Comment:
This and the line below will have to go in a 1.13+ section :)

Reply:
coarse dirt is 1.7